debezium-connector-cassandra/src/test/java/io/debezium/connector/cassandra/SnapshotProcessorTest.java

```diff
@@ -89,17 +91,17 @@ public void testSnapshotEmptyTable() throws Exception {
         context.getCassandraClient().execute("CREATE TABLE IF NOT EXISTS " + keyspaceTable("cdc_table") + " (a int, b text, PRIMARY KEY(a)) WITH cdc = true;");
         context.getSchemaHolder().refreshSchemas();
 
-        BlockingEventQueue<Event> queue = context.getQueue();
-        assertTrue(queue.isEmpty());
+        ChangeEventQueue<Event> queue = context.getQueue();
+        assertTrue(queue.remainingCapacity() == queue.totalCapacity());
```

Comment:
Better use `assertEquals()`; in case of a failure the message will reveal expected and actual values.

Reply:
Updated accordingly~